R/pkg/R/generics.R

```diff
@@ -735,6 +752,8 @@ setGeneric("between", function(x, bounds) { standardGeneric("between") })
 setGeneric("cast", function(x, dataType) { standardGeneric("cast") })
 
 #' @rdname columnfunctions
+#' @param x a Column object.
+#' @param ... additional argument(s).
```

Comment:
It seems the functions are defined in the following way:

column_functions2 <- c("like", "rlike", "getField", "getItem", "contains")

createColumnFunction2 <- function(name) {
  setMethod(name,
            signature(x = "Column"),
            function(x, data) {
              if (class(data) == "Column") {
                data <- data@jc
              }
              jc <- callJMethod(x@jc, name, data)
              column(jc)
            })
}

It seesm the functions are not exported. Perhaps we still leave it there?